src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -279,6 +283,7 @@ class MulticopterPositionControl : public control::SuperBlock
 	math::Vector<3> _vel_ff;
 	math::Vector<3> _vel_sp_prev;
 	math::Vector<3> _vel_err_d;		/**< derivative of current velocity */
+	math::Vector<3> _curr_sp;  /**< the previous current setpoint of the triplets */
```

Comment:
I can infer current setpoint from the name but not that it's a position. Would `_curr_pos_sp` or similar not make more sense?

Why is it previous? If it is previous that should also be in the name.